packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -79,8 +94,8 @@ const ImageGridColumnsTopStory = css`
 const ImageGridColumns = css`
   grid-column: 1 / span 2;
 
-  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
-    grid-column: 1 / span 4;
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    grid-column: none;
```

Comment:
Why do we need this?

Reply:
As we won't need to use grid over 1008px (for normal promos), I am trying to override the grid styles applied to `StoryPromoWrapper`, `ImageGridItem` and `TextGridItem` to none. Do you think this is correct?

Reply:
There is nothing wrong with overriding the grid styles. I was just wondering why you were doing that. Even without overriding it still displays correctly.

Reply:
Yeah, just applying `display:block` to the `StoryPromoWrapper` makes the text sit under the image. Happy to remove the overrides then.